views/templates/block/activityLog.twig

```diff
@@ -19,6 +19,36 @@
             <div id="errorDuringUpgrade" class="col-xs-12" style="display:none;">
                 <h4>{{ 'Errors'|trans({}, 'Modules.Autoupgrade.Admin') }}</h4>
                 <div id="infoError" class="processing"></div>
+                <button id="reportErrorButton" class="btn btn-warning" style="margin-top: 10px;" data-toggle="modal" data-target="#errorModal">
```

Comment:
Clearly not a fan of this `style` attribute, but it seems we have a Bootstrap so old we don't even have the spacing utilities available.

Reply:
I am the first to use utility classes for margins, but not available for this version